packages/datadog-instrumentations/src/opensearch.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const { addHook } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+const { createWrapRequest } = require('./elasticsearch')
+
+addHook({ name: '@opensearch-project/opensearch', file: 'lib/Transport.js', versions: ['1.1.0'] }, Transport => {
```

Comment:
@rochdev In order to get support for 2.x, is the only change needed to add those versions here? From my understanding the transport parts haven't changed so I guess hooking would work the same?

Reply:
Actually this should be a range and not just a single version, so it was probably just changed for testing and I forgot and left it like this. At least all (or most) of v1.x should be supported, and if v2.x didn't change the internals too much it should work as well.